### PR TITLE
Hotfix: Toggling between filters should clear searchText

### PR DESCRIPTION
**Pull request by reshadbinharun**

### CODE REVIEW

File: client/src/components/AlumniDirectory.js [hunk 1]
```diff
@@ -455,7 +460,7 @@ export default class AlumniDirectory extends Component {
             case 'gradYear':
                 return (
                     <Dropdown 
-                        placeholder='Year:'
+                        placeholder={'Graduation Year'}
```
Comment (sbetti02) [inline client/src/components/AlumniDirectory.js:463]: Do the placeholders that are just a string need to be encapsulated as JSX? I think it should be fine to just leave it as `placeholder="Graduation Year"` and `placeholder="Interest"`?

File: server/routes/index.js [hunk 2]
```diff
@@ -329,6 +329,30 @@ router.get('/totalCounts', async (req, res, next) => {
   }
 })
 
+router.get('/sampleSignUps', async (req, res, next) => {
```
Comment (sbetti02) [inline server/routes/index.js:332]: Can you add a line describing what this route is for?

File: client/src/landingPageContent/JoinUs.js [hunk 3]
```diff
@@ -20,10 +22,63 @@ export default function JoinUs (props) {
...
+            <Segment>
+                <Header as='h4'>Get a chance to meet mentors like...</Header>
+                <Grid
+                    stackable
+                    textAlign='center'
+                >
+                    {
+                        sampleSignUps.map(alumnus => {
+                            let transitionDuration = [1000, 2000, 3000][Math.floor(Math.random() * [1000, 2000, 3000].length)]
+                            return (
```
Comment (sbetti02) [inline client/src/landingPageContent/JoinUs.js:50]: It feels like these cards could deserve their own component

**Review by sbetti02 — COMMENTED** [review]
Looks good to me, just a couple minor comments

File: client/src/landingPageContent/JoinUs.js (hunk 3 above)
Comment (reshadbinharun) [reply client/src/landingPageContent/JoinUs.js:53]: I don't think these cards will get reused anywhere else so I don't think that's worth the effort.

File: server/routes/index.js (hunk 2 above)
Comment (reshadbinharun) [reply server/routes/index.js:335]: addressed

**Pull request merged**